### ISSUE TITLE
Plasmaman Cargo paint is missing from spacesuit modification station

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#36038]
(WEB REPORT BY: demagoguedimension REMOTE: 172.93.101.77:7777)
# Revision
9334823d0503a8f9d8bf9baa5ab6c4ba60fa6e52

# Description
See title
# Steps to Reproduce
Go to spacesuit modification station, try to find Cargo Technician or Quartermaster paint
# What you Expected
Those to be available
# What Actually Happened
Those options are not available.

**Comment by Eneocho** [comment]
did you have cargo tech or QM access?

**Comment by Dacendeth** [comment]
Yeah the cargo suit paints are just not available for plasmeme or vox, likely just forgotton

**Comment by Eneocho** [comment]
i'll take your word for it
